mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/forms/general.xml

```diff
@@ -35,6 +35,12 @@
         <type>text</type>
         <help>Port to listen on.</help>
     </field>
+    <field>
+        <id>general.inet_protocol</id>
+        <label>IP Protocol</label>
```

Comment:
It’s actually IP version, not protocol.

Reply:
Or "address family" / "protocol family" would be the term in the bsd socket api :)